docs/src/tutorials/tutorial_5.md

```diff
@@ -0,0 +1,377 @@
+	...
+    ],
+    pathname = "atomic.gif",
+    framerate = 10,
+)
+```
+
+As you can see, the animation we are creating is going to have many frames!
+This is the longest animation we have made so far.
+Why?
+Not only are we going to examine many different elements, this tutorial also serves to illustrate how one can make longer animations to convey ideas.
+Think of it as your directoral debut! 🎬 🎥
+
+## Taming the Elements!
+
+Each element has a different atomic mass.
+This atomic mass is measured in the unit called a "Dalton" (symbol: u) which is equivalent to 1/12 of the mass of a stationary carbon-12 atom.
+We can use the `Scaling` functionality that `Javis.jl` provides to visualize these different masses!
+
+To accomplish this, we need to make a function the shows our currently viewed element:
+
+```julia
+function element(p = O, color = "black")
+    sethue(color)
+    c = circle(p, 1, :fill)
+end
+```
+
+Essentially, all the `element` function does is create a circle in the middle of the frame with the initial size of 1.
+
+From there, we need to define more `Action` objects for our `javis` function for the element we are viewing to scale:
+
+```julia
+...
+        Action(
+            (args...) -> element(),
+            subactions = [
+                SubAction(1:300, Scaling(1, 30)),
+                SubAction(301:350, Scaling(30, 16)),
+            ],
+        ),
+	...
+        pathname = "atomic.gif",
+        framerate = 10
+    )
+...
+```
+
+This will grow our element from `1` to `30` and then shrink the element from `30` to `16`.
+
+That scaling looks like this:
+
+![](assets/blank_atom_scaling.gif)
+
+Staring at this somewhat makes me think of a black hole... ⚫
+But great!
+The only question now is... What are we looking at?
+Let's add some more information to this animation! 📝
+
+## How Much Does an Atom Weigh? ⚖️
+
+To get the information about an element that we are currently previewing, we need to get information about our element.
+The first thing we need to get is the atomic mass of the element.
+So, how do we do that?
+
+First, we must modify our `javis` function slightly to store information about the circle object we draw by using the symbol `:atom`:
+
+```julia
+...
+        Action(
+	    :atom,
+            (args...) -> element(),
+            subactions = [
+                SubAction(1:300, Scaling(1, 30)),
+                SubAction(301:350, Scaling(30, 16)),
+            ],
+        ),
+	...
+```
+
+Now, we must update the `element` function to return the current scale of the circle being drawn.
+The reason why we want the scale of the object is that the scale of the circle represents its atomic mass.
+At this time, `Javis.jl` does not allow a user to get the current scale of an object directly.
+This will most likely be changed in future versions.
+The following changes need to be made to the `element` function:
+
+```julia
+function element(p = O, color = "black")
+    sethue(color)
+    c = circle(p, 1, :fill)
```

Comment:
I think this doesn't return anything and you definitely don't use `c` afterwards so this should be just `circle(p, 1, :fill)`

Reply:
Agreed and fixed. :smile: